### PR TITLE
Integrate User Avatar in Settings #1837, Create getUserAvatar utility #2154

### LINKED ISSUE #1837: Integrate User Avatar in User Account Section with Redux
**Actual Behaviour**

Not present, user should be able to add an avatar, and the avatar should be setup in redux state, so that TopBar and Static Bar bar can use it

**Expected Behaviour**

Integrate User Avatar in User Account Section with Redux

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1837

Changes: 
 - Integrate User Avatar in User Account Section with Redux
 - Created getUserAvatar utility function #2154

Demo Link: https://pr-2152-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@fragm3 I think you have pushed your last PR changes too in this one

File: src/components/Admin/Admin.js [hunk 1]
```diff
@@ -133,9 +133,9 @@ class Admin extends Component {
     return (
       <div>
         <StaticAppBar {...this.props} />
-        {loading ? (
+        {!loading ? (
```
Comment (akshatnitd) [inline src/components/Admin/Admin.js]: Do you want it to `!loading`?

File: src/components/ChatApp/Settings/AccountTab.react.js [hunk 2]
```diff
@@ -32,15 +37,161 @@ const Timezone = styled.div`
...
+    height: '50px',
+    width: '50px',
+  },
+  selectAvatarDropDownStyle: {
+    width: '40%',
+    minWidth: '9.5rem',
+  },
+};
+
+const AvatarRender = props => {
```
Comment (akshatnitd) [inline src/components/ChatApp/Settings/AccountTab.react.js]: Shift it onto a new file

File: src/components/ChatApp/Settings/AccountTab.react.js [hunk 3]
```diff
@@ -108,17 +259,70 @@ class AccountTab extends React.Component {
...
+    let form = new FormData();
+    form.append('access_token', accessToken);
+    form.append('image', file);
+    this.setState({ uploadingAvatar: true });
+    setUserAvatar(form).then(() => {
+      this.setState({
+        uploadingAvatar: false,
+        isAvatarAdded: true,
+        isAvatarUploaded: true,
+      });
```
Comment (akshatnitd) [inline src/components/ChatApp/Settings/AccountTab.react.js]: We again need to call the getAvatar action to see the change reflected on the StaticBar

**Comment by fragm3** [comment]
@akshatnitd conflicts due to branching out, not able to fix, will recreate the PR.

**Pull request closed without merging**